src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorDiagnosticsTests.cs

```diff
@@ -248,7 +248,20 @@ public void DoNotWarnOnClassesWithConstructorInitOnlyProperties()
             CompilationHelper.CheckDiagnosticMessages(DiagnosticSeverity.Warning, generatorDiags, Array.Empty<(Location, string)>());
             CompilationHelper.CheckDiagnosticMessages(DiagnosticSeverity.Error, generatorDiags, Array.Empty<(Location, string)>());
         }
-        
+
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/58770", TestPlatforms.Browser)]
```

Comment:
What's the issue with browser?

Reply:
I honestly am unsure, I copied this from all the other tests in the file.